### DISCUSSION TITLE
Fill a popup after command execution

### DISCUSSION THREAD

**Discussion by Shtooth** [coddingtonbear/obsidian-local-rest-api#136] [Q&A]
Hello guys:

Im trying to use this project to trigger a daily note based on a template created for templater. When I trigger the command, a second input field appears asking what template I want to use. 

Is there any way in which I can provide this information for the command to work?

Is there any way to obtain the information about which fields might be required on a request for similar cases?

Thanks in advance!

**Comment by coddingtonbear** [comment]
At least the last time I looked into this, there was no way to provide additional information as part of calling a command.  That might not be true anymore, though, and I'd love to be corrected if that's no longer the case!
